### ISSUE TITLE
Refactor Docs into it's own repo

### ISSUE THREAD

**Issue by benstoltz** [Esri/cedar#178]
@tomwayson @ajturner: Thoughts on this? I was thinking doing something similar to how esri-leaflet handles it's main repo / docs repo. As we've independently talked about, the tooling currently in place makes maintainability somewhat confusing.

https://github.com/Esri/esri-leaflet
https://github.com/Esri/esri-leaflet-doc

**Comment by ajturner** [comment]
The concern I have with two repos is the difficult keeping them in sync on modifications, updates and tagged releases. 

Alternatively [Chart.JS](http://www.chartjs.org/) uses a completely separate [gh-pages branch](https://github.com/chartjs/Chart.js/tree/gh-pages) and the primary docs maintained as simple [markdown in /docs](https://github.com/chartjs/Chart.js/tree/master/docs)

Foremost in Cedar is cleaning up the various build steps and systems. It evolved somewhat ad-hoc as the project emerged and different people helped out. If we can keep the build clean and the docs simple markdown then it ideally shouldn't be too confusing.

**Comment by tomwayson** [comment]
I agree that priority should be reliable build and release systems and that if we cleaned up what's here, it should be fairly obvious what parts of this repo are concerned with the docs, and what is concerned with the library itself.

Keeping the docs in the same repo/branch as the source code is ultimately going to be the easiest way to manage keeping the versions in sync. If you have to move them out, then I'd prefer another repo than gh-pages branch. I actually think that would be harder. At least w/ a separate docs repo, you could `npm link` to this repo while developing.
